src/server/twemcache/main.c

```diff
@@ -1,3 +1,4 @@
+#include "admin/process.h"
```

Comment:
It was throwing a compilation warning at me when I was running the test. I could exclude this and put it on a separate PR.